src/common/set.c

```diff
@@ -396,38 +391,30 @@ util_map_part(struct pool_set_part *part, void *addr, size_t size,
 	ASSERTeq(size % Mmap_align, 0);
 	ASSERT(((off_t)offset) >= 0);
 
-	void *addrp;
 	if (part->is_dev_dax) {
-		/*
-		 * DAX device can only be in a poolset in which it's the only
-		 * part and we do not need to utilize a hint address.
-		 */
-		addrp = util_map(part->fd, part->filesize, flags, rdonly, 0);
-		if (addrp == MAP_FAILED) {
-			ERR("!mmap: %s", part->path);
-			return -1;
-		}
-		part->addr = addrp;
-		part->size = part->filesize;
-	} else {
-		if (!size)
-			size = (part->filesize & ~(Mmap_align - 1)) - offset;
+		/* ASSERTeq(offset % part->phys_pagesize, 0); */
```

Comment:
we have the new function to retrieve the alignment, so we can do this now.
Maybe include that alignment in the part structure?